src/tox_lsr/config_files/woke.yml

```diff
@@ -3,13 +3,15 @@ ignore_files:
   - .github
   - .sanity-ansible-ignore-*.txt
   - pylintrc
+  - vendor
 
 rules:
   - name: master
     terms:
       - master
     alternatives:
       - primary
+    severity: warning
```

Comment:
no, I think we will need to address this on a case-by-case basis - either with `wokeignore:rule=master` in the file, or with a role local `.wokeignore` that excludes particular files.  Is there some role that just has too many instances of "master"?

Reply:
Ok, I took `severity: warning` back for `master`. It was not an issue of the count of the word `master`, but the fact the word `master` never be able to be eliminated. Following is an example from `nbde_client`. If this is ok, then I'd think we could live with the current severity setting of `master`.

```
diff --git a/library/nbde_client_clevis.py b/library/nbde_client_clevis.py
index 6a359b6..899dc22 100644
--- a/library/nbde_client_clevis.py
+++ b/library/nbde_client_clevis.py
@@ -999,7 +999,8 @@ def unbind_slot(module, device, slot):
 def new_key(module, device):
-    """Generate a new key with the same entropy as the LUKS master key.
+    """wokeignore:rule=master
+    Generate a new key with the same entropy as the LUKS master key.
     Return <key> <error>"""
 ```

Reply:
@richm, Rereading https://source.redhat.com/aboutredhat/weareredhat/inclusion/diversity__inclusion_wiki/conscious_language_project_faq and the rule file of `woke`, I might have set our custom woke rule much tighter than expected. Maybe, just `master/slave` combination (and the `master branch`) must be avoided, I guess...

Reply:
> Ok, I took `severity: warning` back for `master`. It was not an issue of the count of the word `master`, but the fact the word `master` never be able to be eliminated. Following is an example from `nbde_client`. If this is ok, then I'd think we could live with the current severity setting of `master`.
> 
> ```
> diff --git a/library/nbde_client_clevis.py b/library/nbde_client_clevis.py
> index 6a359b6..899dc22 100644
> --- a/library/nbde_client_clevis.py
> +++ b/library/nbde_client_clevis.py
> @@ -999,7 +999,8 @@ def unbind_slot(module, device, slot):
>  def new_key(module, device):
> -    """Generate a new key with the same entropy as the LUKS master key.
> +    """wokeignore:rule=master
> +    Generate a new key with the same entropy as the LUKS master key.
>      Return <key> <error>"""
> ```

Looks like this also works:
```
--- a/library/nbde_client_clevis.py
+++ b/library/nbde_client_clevis.py
@@ -999,6 +999,7 @@ def unbind_slot(module, device, slot):
 
 
 def new_key(module, device):
+    # wokeignore:rule=master
     """Generate a new key with the same entropy as the LUKS master key.
     Return <key> <error>"""
```
Did you try that?  If so, are you concerned that it applies to the entire function, rather than just that line?

Reply:
> @richm, Rereading https://source.redhat.com/aboutredhat/weareredhat/inclusion/diversity__inclusion_wiki/conscious_language_project_faq and the rule file of `woke`, I might have set our custom woke rule much tighter than expected. Maybe, just `master/slave` combination (and the `master branch`) must be avoided, I guess...

I'm not sure what you mean.  The initiative has as the primary goals the elimination, or at least the reduction (if elimination is not possible), of the terms `master/slave` and `whitelist/blacklist`.

Reply:
> > @richm, Rereading https://source.redhat.com/aboutredhat/weareredhat/inclusion/diversity__inclusion_wiki/conscious_language_project_faq and the rule file of `woke`, I might have set our custom woke rule much tighter than expected. Maybe, just the `master/slave` combination (and the `master branch`) must be avoided, I guess...
> 
> I'm not sure what you mean. The initiative has as the primary goals the elimination, or at least the reduction (if elimination is not possible), of the terms `master/slave` and `whitelist/blacklist`.

I got the impression that we are supposed to eliminate "master/slave" or "master-slave" or something "master" and "slave" show up as a pair. Of course, the single word "slave" must be removed. But not the single word "master". At least the default rules that woke provides are configured such as demonstrated next.

In the main branch of the network role:
```
# At least two files refer to "master" at the top level.
$ grep master *
CHANGELOG.md:- `master` is deprecated in favor of `controller`
pylintrc:[MASTER]

# Two files refer to "slave" at the top level.
$ grep slave *
CHANGELOG.md:- `slave` is deprecated in favor of `port`
README.md:    `all_slaves_active` in kernel and NetworkManager. The boolean value `false` drops

# Run woke with the default rules.
# "master" does not get complained, but "slave" does.
$ woke | grep master | wc -l
0
$ woke | grep slave | wc -l
22
```
Yes, this is the default behavior of `woke` and we need to follow the Red Hat policy. And rereading FAQ, it looks to me that the company's policy also supports the same rules that `woke` provides by default (except the suggested words to be replaced with. We want to make these changes: "master/slave" --> "primary/secondary", "whitelist/blacklist" --> "allowlist/blocklist". That's why I've added them to our custom rule file woke.yml".)

Reply:
Ok.  I'm still not sure what you mean here.  Are you suggesting that we relax the rules to allow the use of "master" when it does not occur in a context where it is used explicitly or implicitly with "slave"?

Reply:
> Ok. I'm still not sure what you mean here. Are you suggesting that we relax the rules to allow the use of "master" when it does not occur in a context where it is used explicitly or implicitly with "slave"?

Yes. That's correct.

Reply:
> > Ok. I'm still not sure what you mean here. Are you suggesting that we relax the rules to allow the use of "master" when it does not occur in a context where it is used explicitly or implicitly with "slave"?
> 
> Yes. That's correct.

Ok.  I'm ok with that for now.  Let's proceed with this PR, then revisit "master" at a later date.

Reply:
> Ok. I'm ok with that for now. Let's proceed with this PR, then revisit "master" at a later date.

I've updated this pr once again. Could you please review it? Thanks!